helm-charts/verticadb-operator/values.yaml

```diff
@@ -15,11 +15,10 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
-
-
 image:
   repo: null # null = dockerhub
   name: vertica/verticadb-operator:1.3.1
+  pullPolicy: IfNotPresent # For kind environments this needs to be IfNotPresent
```

Comment:
The comment is misleading.  I suggest we just remove it.  It is best to have that comment in the Makefile, which we already do.